webviz_subsurface/plugins/_structural_uncertainty/controllers/intersection_controller.py

```diff
@@ -319,7 +319,8 @@ def _store_intersection_layout(
             layout.get("yaxis", {}).update(range=user_range)
         # Else autocalculate range if not intersecting a well
         elif intersection_source != "well":
-            del layout.get("yaxis", {})["range"]
+            if "range" in layout.get("yaxis", {}):
+                del layout["yaxis"]["range"]
```

Comment:
Will this work `if "yaxis" not in layout`, som kanskje kan skje siden det brukes `.get(..., {})` over?